### PR TITLE
Storybook: Rewrite stories to CSF

### PR DESCRIPTION
**Pull request by tskarhed**
With the CSF format we get a props table for free! There are a lot of Stories that still needs conversion.

#22958

### CODE REVIEW

File: packages/grafana-ui/src/components/ColorPicker/ColorPicker.story.tsx [hunk 1]
```diff
@@ -1,23 +1,31 @@
...
   };
 };
 
-const ColorPickerStories = storiesOf('Pickers and Editors/ColorPicker/Pickers', module);
-
-ColorPickerStories.addDecorator(withCenteredStory);
+export default {
+  title: 'Pickers and Editors/ColorPicker',
+  component: ColorPicker,
+  subcomponents: { SeriesColorPicker },
```
Comment (dprokop) [inline packages/grafana-ui/src/components/ColorPicker/ColorPicker.story.tsx:19]: what does subcomponent do?

File: packages/grafana-ui/src/components/ColorPicker/ColorPicker.story.tsx [hunk 2]
```diff
@@ -1,23 +1,31 @@
...
+  decorators: [withCenteredStory],
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+};
 
-ColorPickerStories.add('default', () => {
+export const simple = () => {
```
Comment (dprokop) [inline packages/grafana-ui/src/components/ColorPicker/ColorPicker.story.tsx:28]: I wish we could call this `default` :( Maybe `basic` would fit better than simple? Something can be simple but not basic at the same time :)

File: packages/grafana-ui/src/components/ColorPicker/NamedColorsPalette.story.tsx [hunk 3]
```diff
@@ -1,21 +1,30 @@
...
 import { renderComponentWithTheme } from '../../utils/storybook/withTheme';
 import { UseState } from '../../utils/storybook/UseState';
+import mdx from './ColorPicker.mdx';
 
 const BasicGreen = getColorDefinitionByName('green');
 const BasicRed = getColorDefinitionByName('red');
 const LightBlue = getColorDefinitionByName('light-blue');
 
-const NamedColorsPaletteStories = storiesOf('Pickers and Editors/ColorPicker/Palettes/NamedColorsPalette', module);
+// const NamedColorsPaletteStories = storiesOf('Pickers and Editors/ColorPicker/Palettes/NamedColorsPalette', module);
```
Comment (dprokop) [inline packages/grafana-ui/src/components/ColorPicker/NamedColorsPalette.story.tsx:14]:
```suggestion
```

File: packages/grafana-ui/src/components/ColorPicker/NamedColorsPalette.story.tsx [hunk 4]
```diff
@@ -1,21 +1,30 @@
...
 
-NamedColorsPaletteStories.addDecorator(withCenteredStory);
+export default {
+  title: 'Pickers and Editors/ColorPicker/Palettes/NamedColorsPalette',
```
Comment (dprokop) [inline packages/grafana-ui/src/components/ColorPicker/NamedColorsPalette.story.tsx:17]: Would be great if we could have constants for the top most categories for easier manipulation.

File: packages/grafana-ui/src/components/ColorPicker/NamedColorsPalette.story.tsx [hunk 5]
```diff
@@ -56,4 +64,9 @@ NamedColorsPaletteStories.add('Named colors swatch - support for named colors',
       }}
     </UseState>
   );
-});
+};
+
+// NamedColorsPaletteStories.add('Named colors swatch - support for named colors').add(
+//   'Named colors swatch - support for hex values',
+
+// );
```
Comment (dprokop) [inline packages/grafana-ui/src/components/ColorPicker/NamedColorsPalette.story.tsx:72]:
```suggestion
```

**Review by dprokop — APPROVED** [review]
Goodies Sir!

**Review by Clarity-89 — APPROVED** [review]

File: packages/grafana-ui/src/components/ColorPicker/ColorPicker.story.tsx (hunk 2 above)
Comment (tskarhed) [reply packages/grafana-ui/src/components/ColorPicker/ColorPicker.story.tsx:28]: Yeah, basic is probably the way to go at the moment

File: packages/grafana-ui/src/components/ColorPicker/ColorPicker.story.tsx (hunk 1 above)
Comment (tskarhed) [reply packages/grafana-ui/src/components/ColorPicker/ColorPicker.story.tsx:19]: You get more components for the same Prop-table 😄 
image

**Pull request merged**